SpeechCLI/Commands/ModelCommand.cs

```diff
@@ -81,7 +87,15 @@ int OnExecute()
                 else
                 {
                     modelDefinition.ModelKind = "Language";
-                    modelDefinition.Datasets = new List<DatasetIdentity>() { new DatasetIdentity(Guid.Parse(LanguageDataset)) };
+                    if (string.IsNullOrWhiteSpace(PronunciationDataset))
+                    {
+                        modelDefinition.Datasets = new List<DatasetIdentity>() { new DatasetIdentity(Guid.Parse(LanguageDataset)) };
+                    } else
+                    {
+                        modelDefinition.Datasets = new List<DatasetIdentity>() { new DatasetIdentity(Guid.Parse(LanguageDataset)),
```

Comment:
Introducing third property (`PronunciationDataset`) creates potential `ArgumentNullException` in the situation when `LanguageDataset` is null and `PronunciationDataset` is not. This couldn't happen before, because the null check on line 61 implicitly filtered it. Now there needs to be additional condition.

Reply:
Thanks for catching this!